src/Shared/LayoutRenderers/AspNetRequestWebSocketRequestedProtocolsLayoutRenderer.cs

```diff
@@ -28,7 +29,7 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
         {
             // Not available on .NET 3.5
 #if ASP_NET_CORE
-            var websockets = HttpContextAccessor.HttpContext?.WebSockets;
+            var websockets = HttpContextAccessor.HttpContext?.TryGetWebSocket();
```

Comment:
Can `?.` be changed to `.` and have the conditional part in the `TryGetWebSocket`-extension-method ?

Reply:
Changed